### PR TITLE
TF-3312 [iOS] Logout: I got asked permission to use "abcxyz.org" to login

### PR DESCRIPTION
**Pull request by dab246**
## Issue

#3312 

## Resolved

https://github.com/user-attachments/assets/ab330154-f818-4e13-9160-4a1ecf073a39

### CODE REVIEW

File: ios/Runner/Info.plist [hunk 1]
```diff
@@ -17,7 +17,7 @@
 		<key>CFBundleInfoDictionaryVersion</key>
 		<string>6.0</string>
 		<key>CFBundleName</key>
-		<string>TMail</string>
+		<string>TwakeMail</string>
```
Comment (hoangdat) [inline ios/Runner/Info.plist:20]:
```suggestion
		<string>Twake Mail</string>
```

**Review by tddang-linagora — DISMISSED** [review]

**Review by hoangdat — DISMISSED** [review]
please rebase

File: ios/Runner/Info.plist (hunk 1 above)
Comment (dab246) [reply ios/Runner/Info.plist]: done

**Comment by dab246** [comment]
> please rebase

Done

**Review by tddang-linagora — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Pull request merged**